makefile

```diff
@@ -92,5 +93,14 @@ tmp/input/mondo.sssom.tsv: | tmp/input/
 	wget http://purl.obolibrary.org/obo/mondo/mappings/mondo.sssom.tsv -O $@
 
 # creates more than just this file; that goal creates multiple files
-output/medgen_terms_mapping_status.tsv output/obsoleted_medgen_terms_in_mondo.txt: | output/
+output/medgen_terms_mapping_status.tsv output/obsoleted_medgen_terms_in_mondo.txt: tmp/input/mondo.sssom.tsv | output/
 	python src/mondo_mapping_status.py
+
+# ----------------------------------------
+# Robot templates
+# ----------------------------------------
+ftp.ncbi.nlm.nih.gov/pub/meedgen/MedGenIDMappings.txt: ftp.ncbi.nlm.nih.gov
```

Comment:
That is a bit weird a dependency? `ftp.ncbi.nlm.nih.gov`?

Reply:
It's not so weird but kinda weird.

Firstly, `ftp.ncbi.nlm.nih.gov` is a directory. I just made it clearer by adding a `/` to it in the goal name and in this dependency usage.

This goal was created by Chris initially, and it downloads the FTP files to this directory. Once that goal is done, `ftp.ncbi.nlm.nih.gov/pub/meedgen/MedGenIDMappings.txt` will already exist. That's why this goal doesn't have any steps.

So, instead of `ROBOT_addMedGen_fromIngest.tsv: ftp.ncbi.nlm.nih.gov/pub/medgen/MedGenIDMappings.txt`, I could have done `ROBOT_addMedGen_fromIngest.tsv: ftp.ncbi.nlm.nih.gov/`, but I kind of like the way I set it up. Makes it more explicit, and will also properly error early if the name of that file or its path ever changes (or if it gets deleted).